.github/workflows/deploy.yml

```diff
@@ -28,12 +28,23 @@ jobs:
         shell: bash
         run: ./scripts/build.sh
 
-      - name: Deploy 🚀
+      - name: Deploy Developer Documentation 🚀
         uses: JamesIves/github-pages-deploy-action@v4.4.1
         with:
           branch: gh-pages
           folder: build
           # Prevent the default behavior of force pushing which will remove previews see https://github.com/marketplace/actions/deploy-pr-preview.
           force: false
           clean: true
-          clean-exclude: pr-preview/
+          clean-exclude: |
+            pr-preview/
+            pr-preview-academy/
+
+      - name: Deploy Concordium Academy 🚀
+        uses: JamesIves/github-pages-deploy-action@v4.4.1
+        with:
+          branch: gh-pages
+          repository-name: Concordium/concordium-academy
+          ssh-key: ${{ secrets.CONCORDIUM_ACADEMY_DEPLOY_KEY }}
```

Comment:
Where is it documented that this needs to be managed?

Could we have it on Confluence perhaps? Or in a README?

Reply:
I have added a section in the Readme of this repo